README.md

```diff
@@ -2,6 +2,67 @@
 
 Run websites as if they were apps.
 
+## Contents
+- [Dependencies](#dependencies)
+- [How to Build/install](#how-to-build/install)
+	- [Debian/Ubuntu based systems](#debian/ubuntu-based-systems: "Debian/Ubuntu based systems")
+	- [Other Unix-like systems](#other-unix-like-systems: "Other Unix-like systems:")
+- [FAQ](#faq)
+	- [How to go back without the navigation buttons?](#how-to-go-back-without-the-navigation-buttons?)
+	- [How to open links in my main browser?](#how-to-open-links-in-my-main-browser?)
+	- [How to use tabs in Firefox?](#how-to-use-tabs-in-firefox?)
+	- [How to add extensions in Firefox (AdBlock etc.)?](#how-to-add-extensions-in-firefox-(adblock-etc.)?)
+	- [How to add extensions in Chromium based browsers (AdBlock etc.)?](#how-to-add-extensions-in-chromium-based-browsers-(adblock-etc.)?)
+
+## Dependencies
+```
+gir1.2-xapp-1.0 (>= 1.4)
+python3
+python3-bs4
+python3-configobj
+python3-gi
+python3-setproctitle
+python3-tldextract
+xapps-common
+```
+
+## How to Build/install
+### Debian/Ubuntu based systems:
+1. Install dependencies:
+	``` 
+	sudo apt install gir1.2-xapp-1.0 python3 python3-bs4 python3-configobj \
+	python3-gi python3-setproctitle python3-tldextract xapps-common
+	```
+
+2. There are two methods, this app can be installed/used:
+	1. **Option 1:** Manually copy necessary files to root (`/`)
+		1. [**Optional**] To make translations/locales in languages other than **English**, run:
+			```
+			make
+			```
+			from the `/path/to/repo` in a terminal. It will create the translations/locales in `usr/share/locale`.
+		
+		2. Copy the contents of `etc` and `usr` to `/etc/` and `/usr/` respectively:
+			```
+			sudo cp -R usr etc /
+			```
+		3. Compile `schemas` using:
+			```
+			glib-compile-schemas /usr/share/glib-2.0/schemas
```

Comment:
This might need to be run with `sudo`. I'm not sure, though.

Reply:
it does